exporter/clickhouseexporter/exporter.go

```diff
@@ -28,30 +29,31 @@ import (
 )
 
 type clickhouseExporter struct {
-	client        *sql.DB
-	insertLogsSQL string
+	client          *sql.DB
+	insertLogsSQL   string
+	insertTracesSQL string
 
 	logger *zap.Logger
 	cfg    *Config
 }
 
-func newExporter(logger *zap.Logger, cfg *Config) (*clickhouseExporter, error) {
+func newExporter(logger *zap.Logger, cfg *Config, createLogsTable, createTracesTable bool) (
+	*clickhouseExporter, error) {
 	if err := cfg.Validate(); err != nil {
 		return nil, err
 	}
 
-	client, err := newClickhouseClient(cfg)
+	client, err := newClickhouseClient(cfg, createLogsTable, createTracesTable)
```

Comment:
I believe it'd be cleaner if we have different factories `newLogsExporter` and `newTraceExporter` instead of one `newExporter`. Each of them creates databases after calling `newClickhouseClient`. `newClickhouseClient` should just create client, not databases

Reply:
Thank you , I have refactored the exporter factories, please review.

Reply:
hi @dmitryax, I have refactored the exporter factories, please review, thank you.